parser/parser.go

```diff
@@ -116,7 +119,16 @@ func (parser *Parser) CheckRealPackagePath(packagePath string) string {
 	pkgRealpath := ""
 	vendoringEnabled := os.Getenv("GO15VENDOREXPERIMENT")
 	if vendoringEnabled == "1" {
-		if evalutedPath, err := filepath.EvalSymlinks(filepath.Join("vendor", packagePath)); err == nil {
+		// evaluate if the user specified a different vendor directory rather
```

Comment:
This isn't going to work on Go 1.7 and above.  

See https://golang.org/doc/go1.7#cmd/go.  Vendoring cannot be turned off and the 

GO15VENDOREXPERIMENT is no longer used.  It may work locally for you as you still have GO15VENDOREXPERIMENT in your environment but in won't work for everyone.